picard/formats/id3.py

```diff
@@ -579,15 +580,17 @@ def _remove_deleted_tags(self, metadata, tags):
                             del tags[key]
                 elif real_name in self.__translate:
                     del tags[real_name]
+                elif name.lower() in self.__rtranslate_freetext_ci:
```

Comment:
Just a minor unrelated thing, I just saw, in the code above we have:

```python
  for key, frame in list(tags.items()):
      if frame.FrameID == 'POPM' and frame.email == config.setting['rating_user_email']:
          del tags[key]
```

Imho it should be rewritten to:

```python
  user_email = config.setting['rating_user_email']
  for key, frame in tags.items():
      if frame.FrameID == 'POPM' and frame.email == user_email:
          del tags[key]
```